tests/pathfinding/test_api.py

```diff
@@ -722,3 +722,16 @@ def test_suggest_partner_api(api_url: str, token_network_model: TokenNetwork):
     url = api_url + f"/v1/{token_network_address_hex}/suggest_partner"
     response = requests.get(url)
     assert response.status_code == 200
+
+
+@pytest.mark.usefixtures("api_sut")
+def test_online_addresses(api_url: str, reachability_state):
+    online_addr = make_address()
+    reachability_state.reachabilities = {
+        online_addr: AddressReachability.REACHABLE,
+        make_address(): AddressReachability.UNREACHABLE,
+    }
+    url = api_url + "/v1/online_addresses"
+    response = requests.get(url)
+    assert response.status_code == 200
+    assert response.json() == [to_checksum_address(online_addr)]
```

Comment:
I tend to want to have `n>=2` examples for tests involving a "many" thing, but I guess this is easy enough...